docs/docs/02-Features/02-Security/auto-invalidate-secrets.md

```diff
@@ -27,7 +27,7 @@ To make this automation opperational, following Azure Resources has to be used:
 
 Our background job has to be configured in `ConfigureServices` method:
 
-```csharp
+```objectivec
```

Comment:
Are these on purpose? If not, can you please revert it to bring our C# syntax back?

Reply:
It is, as far as I know, cause using `objectivec` adds syntax highlighting, `csharp` does not.

Reply:
But it is not using the correct syntax highlighting though 🤔

Reply:
But perhaps this is a way the `csharp` line can stay?
https://github.com/FormidableLabs/prism-react-renderer#faq

Reply:
This might do the trick:
https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages

Reply:
@THMKAE, Thanks that works.